sycl/include/CL/sycl/half_type.hpp

```diff
@@ -275,23 +460,23 @@ template <> struct numeric_limits<cl::sycl::half> {
   static constexpr bool is_iec559 = true;
   static constexpr float_round_style round_style = round_to_nearest;
 
-  static __SYCL_CONSTEXPR_ON_DEVICE const cl::sycl::half(min)() noexcept {
+  static __SYCL_HALF_CTOR_CONSTEXPR const cl::sycl::half(min)() noexcept {
```

Comment:
This doesn't look like a constructor, would it be better to rename `__SYCL_HALF_CTOR_CONSTEXPR` into `__SYCL_HALF_CONSTEXPR`?